app/src/main/res/layout/onboarding_slide.xml

```diff
@@ -20,7 +20,7 @@
       android:layout_height="0dp"
       android:contentDescription="@{viewModel.contentDescription}"
       android:src="@{viewModel.slideImage}"
-      app:layout_constraintDimensionRatio="V,4:4"
+      app:layout_constraintDimensionRatio="1:.8"
```

Comment:
Instead of this mention `5:4` as it is more commonly know ratio for images as compared to `1:0.8`

Reply:
Done